Geode.Angular/src/app/app.component.ts

```diff
@@ -1,17 +1,13 @@
 import { Component, OnInit } from '@angular/core';
-import { AuthService } from './auth/auth.service';
+import { AuthService } from './pages/auth/auth.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrl: './app.component.css'
+  styleUrl: './app.component.css',
 })
-export class AppComponent implements OnInit {
+export class AppComponent {
   title = 'Geode';
 
-  constructor(private authService: AuthService){}
-
-  ngOnInit(): void {
-    this.authService.updateUserAuthorizationInfo()
-  }
+  constructor(private authService: AuthService) {}
```

Comment:
You can remove it totally, and leave empty:
`export class AppComponent {}`

Reply:
I'm using this service for sidebar showing logic in the 'feature-sidebar' branch
Should I still remove it in this branch?